app/components/game-card.js

```diff
@@ -30,6 +36,30 @@ export default Ember.Component.extend({
   /** @property {Boolean} only interact with the cards if you're player one */
   readOnly: false,
 
+  /** Attach document key handlers for magnifying purposes. */
+  attachKeyHandlers: Ember.on('init', function () {
```

Comment:
haven't tested this yet but do you think implementing the keyDown and keyUp actions directly on the component would prevent you from having to inspect whether or not you clicked down on the game-card?